api/controllers/places.js

```diff
@@ -52,6 +56,12 @@ const fetchHydratedDoc = (id, callback) => {
     if (err) {
       return callback(err);
     }
+
+    if (!lineage.rows.length) {
+      return callback({statusCode: 404});
```

Comment:
aping an old api expectation. Might actually be worth keeping and formalising in other lineage code